ubyssey/widgets.py

```diff
@@ -128,15 +139,24 @@ class FacebookVideoBig(Widget):
     end_time = DateTimeField('End Time')
 
     def context(self, result):
-        today = datetime.today()
-        do_show = True
 
-        if result['start_time'] and today < result['start_time'].replace(tzinfo=None):
-            do_show = False
+        result['do_show'] = in_date_range(result['start_time'], result['end_time'])
+        return result
 
-        if result['end_time'] and today > result['end_time'].replace(tzinfo=None):
-            do_show = False
+@register.widget
+class LiveVideoBanner(Widget):
```

Comment:
I think we should call this `AlertBanner` to show that it can be used for generic messages.